src/app/components/Forms/EditForm.tsx

```diff
@@ -0,0 +1,76 @@
+import React, { useState } from 'react'
+import FormInput from '../FormInput/FormInput'
+import PopupForm from './PopupForm'
+
+type ContentType = {
+  title?: string
+  description?: string
+}
+
+type EditFormProps = {
+  onSubmit: (payload: ContentType) => void
+  onCancel: () => void
+  content: ContentType
+}
+
+const MAX_LENGTHS = {
+  need: 80,
+  title: 40,
+  description: 144,
+}
+
+type DEFAULT = {
+  description: string
+}
+
+export default function EditForm({
+  onSubmit,
+  onCancel,
+  content,
+}: EditFormProps): JSX.Element {
+  const [data, setData] = useState<ContentType | DEFAULT>(content)
+
+  function handleSubmit(event: React.FormEvent<HTMLFormElement>) {
+    event.preventDefault()
+
+    data && onSubmit(data)
+  }
+
+  function handleCancel() {
+    setData({ description: '' })
+    onCancel()
+  }
+
+  const ContentKeys = Object.keys(data)
+
+  return (
+    <PopupForm
+      SubmitText="Edit"
```

Comment:
It's more common to have camelCase here

```suggestion
      submitText="Edit"
```